frontend/packages/topology/src/components/list-view/TopologyListViewKindGroup.tsx

```diff
@@ -32,21 +34,21 @@ const TopologyListViewKindGroup: React.FC<TopologyListViewKindGroupProps> = ({
   childNodes.sort((a, b) => a.getLabel().localeCompare(b.getLabel()));
 
   return (
-    <DataListItem key={kind} aria-labelledby={`${kind}_label`} isExpanded>
+    <DataListItem key={kind} aria-label={`${kind}_label`} isExpanded>
```

Comment:
Hey @debsmita1, `aria-label` and `aria-labelledby` have a different meaning. The first one should explain something to the user, the second one refers a dom element which does that. So it's fine to have an ID-like attribute in aria-labelledby, but not in `aria-label`.

So if we have a label somewhere we should link it. For example if it's a list of Deployments we should link the Deployment headline above it. Thats the reason why `aria-labelledby`  is used here before.